### ISSUE TITLE
`public_ip_pc` is missing from Greybel intrinsics space

### ISSUE THREAD

**Issue by Olipro** [ayecue/greybel-vs#142]
In GH, there is a global intrinsic called `public_ip_pc` - this is an alias of the `public_ip` key that is inserted into `computer` maps.

Why use this, you wonder?

I retrieve what is supposed to be a computer object from `get_custom_object` - but I don't trust the process that launched me, so instead of invoking intrinsics by accessing keys on the passed object, I directly call the intrinsics since they're guaranteed to fail if the object is "evil"

**Comment by ayecue** [comment]
It seems like all GreyHack intrinsics are exposed within the global scope not just via the their map. Somehow I never realized that. I'll need to add all of them to the global scope as well.

**Comment by ayecue** [comment]
With 1.10.0 - https://github.com/ayecue/greybel-vs/pull/143 merged I've exposed all GreyHack intrinsics to the global scope now including `public_ip_pc`.

**Comment by Olipro** [comment]
> It seems like all GreyHack intrinsics are exposed within the global scope not just via the their map. Somehow I never realized that. I'll need to add all of them to the global scope as well.

Yeah, I actually have my suspicion this is why the author of GH deleted the `version` intrinsic - he's been chucking everything into global space, which causes a conflict.

Anyhow, all good, thanks.
